samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -271,6 +376,129 @@ static uint8_t const dtm_prbs_content[] = {
 	0x8A, 0x84, 0x39, 0xF4, 0x36, 0x0B, 0xF7
 };
 
+const uint32_t nrf_power_value[] = {
```

Comment:
Consider moving this array to dtm_hw.c and add an API for accessing it

Reply:
Moved